src/toaster/stories/index.stories.js

```diff
@@ -120,6 +121,51 @@ storiesOf('toaster', module).add('examples', () => (
           Danger with Text
         </Button>
       </Box>
+      <Box marginBottom={12}>
+        <Button
+          marginRight={8}
+          onClick={() =>
+            toaster.notify('Messages now is on left bottom notify', {
+              position: AbsolutePosition.BOTTOM_LEFT
+            })
+          }
+        >
+          Left bottom
+        </Button>
+
+        <Button
+          marginRight={8}
+          onClick={() =>
+            toaster.notify('Messages now is on right bottom notify', {
+              position: AbsolutePosition.BOTTOM_RIGHT
+            })
+          }
+        >
+          Right bottom
+        </Button>
+
+        <Button
+          marginRight={8}
+          onClick={() =>
+            toaster.notify('Messages now is on top left message', {
+              position: AbsolutePosition.TOP_LEFT
```

Comment:
Does it make sense for each `toaster.notify` call to require the position? I'm wondering if instead you pass a default value to Evergreen using the theme provider maybe?

Reply:
I believe that pass for each `toaster.notify` it's more easy because if we pass default value to Evergreen and in any moment we need of notification in other place, it's will be not possible.